src/engine/source/base/src/utils/timeUtils.cpp

```diff
@@ -34,6 +35,14 @@ std::string getCurrentTimestamp()
     return getTimestamp(std::time(nullptr));
 }
 
+std::string getCurrentDate(const std::string separator = "/")
```

Comment:
Use a string& or a string_view to prevent copies
```suggestion
std::string getCurrentDate(const std::string_view separator = "/")
```

Reply:
Dang, I forgot the &. Thanks